flink-connectors/flink-connector-rabbitmq/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,9 @@
+flink-connector-rabbitmq
+Copyright 2014-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- com.rabbitmq:amqp-client:4.2.0
```

Comment:
We don't need the file for this dependency for two reasons:
1. We do not bundle the dependency. We need to create the NOTICE file if we e.g. shade the dependency into an uber-jar. In this case user has to manually add/bundle the dependency for the consumer to work.
2. It is ASL license.

Reply:
Ok great, thank you! Are the changes I listed in the source files themselves good enough for ASL license-compliance? Never modified source into a project of this scale before...

Reply:
Yes, the classes itself are good, imo.